File: Jongryul/01/9625.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "input.txt";
const k = Number(fs.readFileSync(filePath).toString().trim());

const dp = [
  [1, 0],
  [0, 1],
];

for (let i = dp.length - 1; i < k + 1; i++) {
  dp.push([dp[i][0] + dp[i - 1][0], dp[i][1] + dp[i - 1][1]]);
}
console.log(dp[k][0], dp[k][1]);

File: Jongryul/01/9625.py
k = int(input())

dp = [[1, 0], [0, 1]]
for i in range(len(dp)-1, k+1):
	dp.append([dp[i][0] + dp[i-1][0], dp[i][1] + dp[i-1][1]])
print(*dp[k])